packages/ndla-ui/src/locale/messages-en.ts

```diff
@@ -1195,11 +1195,11 @@ const messages = {
       drawerButton: 'Show folders and resources',
       drawerTitle: 'Folders and resources',
       description: {
-        all: 'In this folder you find articles and tasks from NDLA. The articles have been collected and placed in order by a teacher.\n\nYou can use the menu to navigate through the articles.\n\nIf you want to come back to the folder later, you can use the link, or you can bookmark the page.',
+        all: 'In this folder you find articles and tasks from NDLA. The articles have been collected and placed in order by a teacher.\n\nYou can use the menu to navigate through the articles.\n\nIf you want to come back to the folder later, you can use the link you received or bookmark the page.',
```

Comment:
folder `you [can/will]`

Reply:
Om ikke det var en tekstendring de spurte om 🤷🏼‍♀️

Reply:
Dette er nok etterspurt av NDLA!

Reply:
will tenker eg er fint.